sentinelPerfTest.d

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Shouldn't this file be part of the benchmark suite?

Reply:
I wasn't planning on keeping it in the finalized change.  Where is the benchmark suite?  Maybe it would make sense to add this to it.

Reply:
In `/benchmark`

Reply:
(though that one is specifically about comparing different druntime versions, but I think just moving the file into this folder should be good enough as it will be a lot easier to find when it's checked in the source code.)